.github/workflows/integration_test.yaml

```diff
@@ -12,4 +12,8 @@ jobs:
       juju-channel: 3/stable
       self-hosted-runner: true
       charmcraft-channel: latest/edge
-      modules: '["test_action.py", "test_config.py", "test_charm.py", "test_http_interface.py", "test_cos.py", "test_ingress.py", "test_ha.py"]'
+      modules: '["test_action.py", "test_config.py", "test_charm.py", "test_http_interface.py", "test_cos.py", "test_ingress.py", "test_ha.py", "test_haproxy_route.py"]'
+      pre-run-script: |
+        -c "curl -L https://github.com/canonical/haproxy-operator/releases/download/rev136/any-charm_ubuntu-22.04-amd64.charm -o ${GITHUB_WORKSPACE}/any-charm_ubuntu-22.04-amd64.charm"
```

Comment:
Can't we fetch any-charm from charmhub?

Reply:
Yes, this was here while waiting for the latest revision to be published, I removed it.